tensorflow/lite/kernels/neg.cc

```diff
@@ -58,10 +104,18 @@ TfLiteStatus Eval(TfLiteContext* context, TfLiteNode* node) {
                             GetTensorShape(output),
                             GetTensorData<float>(output));
       break;
+    case kTfLiteInt8: {
+      auto op_data = static_cast<OpData*>(node->user_data);
+      reference_integer_ops::Negate(
+          GetTensorShape(input), GetTensorData<int8_t>(input),
+          GetTensorShape(output), GetTensorData<int8_t>(output),
+          op_data->zero_points_sum);
+    } break;
```

Comment:
`third_party/tensorflow/lite/micro/kernels/neg.cc:113:54: error: missing field 'profiling_string' initializer [-Werror,-Wmissing-field-initializers]
                                 /*invoke=*/neg::Eval}`

Reply:
please fix above error

Reply:
Fixed.
Also, -Wmissing-field-initializers is not enabled here.